influxdb/docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: "3.7"
+
+services:
+  
+  app_proxy:
+    environment:
+      APP_HOST: influxdb_influxdb_1
+      APP_PORT: 8886
+      PROXY_AUTH_WHITELIST: "/api/*"
+      
+  influxdb:
+    image: influxdb:1.11.7@sha256:b5efc528ee424e37716ce4596d32ad74cb595a4014741b25860e24feacb9868f
+    user: 1000:1000
+    restart: on-failure
+    stop_grace_period: 1m
+    volumes:
+      - ${APP_DATA_DIR}/data/data:/var/lib/influxdb
+      - ${APP_DATA_DIR}/data/config:/etc/influxdb
+      - ${APP_DATA_DIR}/data/influxdb:/.influxdb
+    environment:
+      V1_AUTH_USERNAME: admin
+      V1_AUTH_PASSWORD: admin

```

Comment:
Are these from some legacy version with a UI? Or maybe for the api? As shown above, I can ping the API on port 8086.